Cargo.toml

```diff
@@ -278,3 +278,4 @@ urlencoding = "2.1.2"
 webbrowser = "0.8.7"
 which = "4.4.0"
 unicode-segmentation = "1.10.1"
+notify-debouncer-full = "0.3.1"
```

Comment:
`notify-debouncer`reexports `notify` and recommend to use it unless custom feature is needed, which works for us.